src/package.lisp

```diff
@@ -60,9 +60,17 @@
    #:ed25519-key-x #:ed25519-key-y
 
    ;; pseudo-random number generators
-   #:pseudo-random-number-generator #:list-all-prngs #:make-prng #:random-data
-   #:read-os-random-seed #:read-seed #:write-seed #:fortuna-prng
-   #:add-random-event #:fortuna #:strong-random #:random-bits #:*prng*
+   #:list-all-prngs #:make-prng #:random-data #:read-os-random-seed
```

Comment:
I really wanted to remove `READ-OS-RANDOM-SEED` too, but it was exported before and I've tried to be as conservative as possible — but no more — in these changes.